common/djangoapps/lang_pref/views.py

```diff
@@ -0,0 +1,22 @@
+"""
+Language Preference Views
+"""
+import json
+from django.conf import settings
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.translation import LANGUAGE_SESSION_KEY
+from lang_pref import LANGUAGE_KEY
+from django.http import HttpResponse
+
+
+@csrf_exempt
```

Comment:
what is the reason for `@csrf_exempt` here I believe we are sending CSRFToken for this.